### ISSUE TITLE
Issue in week 5

### ISSUE THREAD

**Issue by KunjManiar** [ravibitsgoa/Parallelism-Intel-Arch-Assignments#6]
The week5 worker.cc takes more than 0.9sec to give output thus results in 0 marks

**Comment by arunkumaraqm** [comment]
No, it takes 0.5-0.7 seconds. If the execution times out, your send and recv calls are likely causing a deadlock. If there are errors, then you should add this flag to the `mpirun` command: `-env I_MPI_SHM_LMT=shm`
It looks like this repo was just uploaded on August 2019 but @ravibitsgoa might have taken the course long before that. The lab files changed in Jan 2019; you should change the `phase` to a constant 0.5f in worker.cc.
